src/libraries/System.Text.Json/tests/Utf8JsonReaderTests.MultiSegment.cs

```diff
@@ -899,6 +899,60 @@ public static void TestSingleStringsMultiSegment()
             }
         }
 
+        [Fact]
+        public static void TestMultiSegmentStringConversionToDateTime()
+        {
+            string jsonString = "\"1997-07-16\"";
+            string expectedString = "1997-07-16";
+
+            byte[] dataUtf8 = Encoding.UTF8.GetBytes(jsonString);
+
+            ReadOnlySequence<byte> sequence = JsonTestHelper.CreateSegments(dataUtf8);
+
+            var json = new Utf8JsonReader(sequence, isFinalBlock: false, state: default);
```

Comment:
nit: Consider looping through the entire sequence and creating slices of arbitrary lengths (and re-entering the reader with "partial data"). See other "single value" tests like: `TestSingleStringsMultiSegment`

Reply:
Tried something out with 823f147, let me know if that covers what you were thinking.